modules/core/src/main/java/org/apache/ignite/internal/metric/IoStatisticsHolderCache.java

```diff
@@ -48,12 +50,23 @@
 
     /**
      * @param cacheName Name of cache.
+     * @param grpId Group id.
+     * @param mreg Metric registry.
      */
-    public IoStatisticsHolderCache(String cacheName, int grpId) {
+    public IoStatisticsHolderCache(String cacheName, int grpId, MetricRegistry mreg) {
         assert cacheName != null;
 
         this.cacheName = cacheName;
         this.grpId = grpId;
+
+        MetricRegistry mset = mreg.withPrefix(CACHE_GROUP.metricGroupName(), cacheName);
```

Comment:
Variable names mset and mreg doesn't follow code style.
Use names registry and locRegistry instead.

Reply:
Why is that? Can you provide a rule I violate with these names?

Reply:
I think mreg, mset is not very descriptive, check RULE2 of abbreviation rules.
reg or registry are more descriptive variants for MetricRegistry instance.

Reply:
mset doesn't look very descriptive, see RULE2 of abbreviation rules.